deployments/types/common.json

```
@@ -0,0 +1,134 @@
+  "MillauBlockNumber": "u64",
+  "MillauHeader": {
+    "parent_Hash": "MillauBlockHash",
+    "number": "Compact<MillauBlockNumber>",
+    "state_root": "MillauBlockHash",
+    "extrinsics_root": "MillauBlockHash",
+    "digest": "MillauDigest"
+  },
+  "MillauDigest": {
+    "logs": "Vec<MillauDigestItem>"
+  },
+  "MillauDigestItem": {
+    "_enum": {
+      "Other": "Vec<u8>",
+      "AuthoritiesChange": "Vec<AuthorityId>",
+      "ChangesTrieRoot": "MillauBlockHash",
+      "SealV0": "SealV0",
+      "Consensus": "Consensus",
+      "Seal": "Seal",
+      "PreRuntime": "PreRuntime"
+    }
+  },
+  "--2": "Rialto Types",
+  "RialtoBalance": "u128",
+  "RialtoBlockHash": "H256",
+  "RialtoBlockNumber": "u32",
+  "RialtoHeader": {
+    "parent_Hash": "RialtoBlockHash",
+    "number": "Compact<RialtoBlockNumber>",
+    "state_root": "RialtoBlockHash",
+    "extrinsics_root": "RialtoBlockHash",
+    "digest": "RialtoDigest"
+  },
+  "RialtoDigest": {
+    "logs": "Vec<RialtoDigestItem>"
+  },
+  "RialtoDigestItem": {
+    "_enum": {
+      "Other": "Vec<u8>",
+      "AuthoritiesChange": "Vec<AuthorityId>",
+      "ChangesTrieRoot": "RialtoBlockHash",
+      "SealV0": "SealV0",
+      "Consensus": "Consensus",
+      "Seal": "Seal",
+      "PreRuntime": "PreRuntime"
+    }
+  },
+  "--3": "Common types",
+  "Address": "AccountId",
+  "LookupSource": "AccountId",
+  "AccountSigner": "MultiSigner",
+  "SpecVersion": "u32",
+  "RelayerId": "AccountId",
+  "SourceAccountId": "AccountId",
+  "ImportedHeader": {
+    "header": "BridgedHeader",
+    "requires_justification": "bool",
+    "is_finalized": "bool",
+    "signal_hash": "Option<BridgedBlockHash>"
+  },
+  "AuthoritySet": {
+    "authorities": "AuthorityList",
+    "set_id": "SetId"
+  },
+  "Id": "[u8; 4]",
+  "InstanceId": "Id",
+  "LaneId": "Id",
+  "MessageNonce": "u64",
+  "MessageId": "(Id, u64)",
+  "MessageKey": {
+    "lane_id": "LaneId",
+    "nonce:": "MessageNonce"
+  },
+  "InboundRelayer": "AccountId",
+  "InboundLaneData": {
+    "relayers": "Vec<(MessageNonce, MessageNonce, RelayerId)>",
+    "last_confirmed_nonce": "MessageNonce"
+  },
+  "OutboundLaneData": {
+    "latest_generated_nonce": "MessageNonce",
+    "latest_received_nonce": "MessageNonce",
+    "oldest_unpruned_nonce": "MessageNonce"
+  },
+  "MessageData": {
+    "payload": "MessagePayload",
+    "fee": "Fee"
+  },
+  "MessagePayload": "Vec<u8>",
+  "BridgedOpaqueCall": "Vec<u8>",
+  "OutboundMessageFee": "Fee",
+  "OutboundPayload": {
+    "spec_version": "SpecVersion",
+    "weight": "Weight",
+    "origin": "CallOrigin",
+    "call": "BridgedOpaqueCall"
+  },
+  "CallOrigin": {
+    "_enum": {
+      "SourceRoot": "()",
+      "TargetAccount": "(SourceAccountId, MultiSigner, MultiSignature)",
+      "SourceAccount": "SourceAccountId"
+    }
+  },
+  "MultiSigner": {
+    "_enum": {
+      "Ed25519": "H256",
+      "Sr25519": "H256",
+      "Ecdsa": "[u8;33]"
+    }
+  },
+  "MesssagesProofOf": {
```

Is there something wrong with the `s` key on your keyboard @tomusdrw 🤣 

```suggestion
  "MessagesProofOf": {
```

---

That's insane, I feel I fixed this dozen of times alraedy and it keeps coming back :D